src/components/comment/comment.scss

```diff
@@ -50,6 +51,10 @@
 
 .c-comment.is-replying .c-comment__reply-form {
   display: block;
+  /// The top margin matches the grid gap spacing applied to the comment form.
+  /// Otherwise, the form's help text appears visually closer to the comment
+  /// you're responding to than to the form.
+  margin-top: ms.step(2, 1rem);
```

Comment:
Did you consider using what's used for the replies themselves to inherit any rhythm object spacing?

```
margin-top: var(--rhythm, #{size.$rhythm-default});
```

Reply:
This is what I was doing previously, but the rhythm spacing between replies is smaller than the spacing within the comment form. 

This led to the help text being closer to the comment above it than to the rest of the form. I worried that made it less clear what the help text was connected to.

Though maybe if the rhythm spacing used on the WP site is larger that won't be an issue?

Reply:
In that case, can we use whatever token the form is using instead of a new number? (And if it's not using a token, maybe it should be?)

Reply:
Sure, I can switch the form to using a token and use it here as well.